### PR TITLE
doc: add document for argocd

### PR DESCRIPTION
**Pull request by yangchiu**
doc: add document for argocd

For https://github.com/longhorn/longhorn/issues/7360

Signed-off-by: Yang Chiu <yang.chiu@suse.com>

### CODE REVIEW

File: content/docs/1.6.0/deploy/install/install-with-argocd.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    kubectl apply -f longhorn-application.yaml
+    ```
+
+4. Sync (Deploy) the Application.
+
+    ```bash
+    argocd app sync longhorn
+    ```
+
+    This command will synchronize the Longhorn Application, deploying Longhorn with the specified configurations.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-argocd.md:66]: Please remove this. If users want a more general description of the `argocd app sync` command, they can read the Argo CD documentation.

File: content/docs/1.6.0/deploy/install/install-with-argocd.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+5. Verify that Longhorn was installed successfully.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-argocd.md:68]:
```suggestion
1. Verify that Longhorn was installed successfully.
```

File: content/docs/1.6.0/deploy/install/install-with-argocd.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+    csi-resizer-6d8cf5f99f-68cw7                        1/1     Running   0          101s
+    csi-attacher-7bf4b7f996-df8v6                       1/1     Running   0          101s
+    csi-attacher-7bf4b7f996-g9cwc                       1/1     Running   0          101s
+    csi-attacher-7bf4b7f996-8l9sw                       1/1     Running   0          101s
+    csi-resizer-6d8cf5f99f-smdjw                        1/1     Running   0          101s
+    instance-manager-b34d5db1fe1e2d52bcfb308be3166cfc   1/1     Running   0          114s
+    engine-image-ei-df38d2e5-cv6nc                      1/1     Running   0          114s
+    ```
+
+6. [Create an NGINX Ingress controller with basic authentication](../../accessing-the-ui/longhorn-ingress) to access the Longhorn UI. Authentication to the Longhorn UI is not enabled by default.
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-argocd.md:98]:
```suggestion
1. [Create an NGINX Ingress controller with basic authentication](../../accessing-the-ui/longhorn-ingress) to access the Longhorn UI. Authentication to the Longhorn UI is not enabled by default.
```

File: content/docs/1.6.0/deploy/install/install-with-argocd.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+7. [Access the Longhorn UI](../../accessing-the-ui).
```
Comment (jillian-maroket) [inline content/docs/1.6.0/deploy/install/install-with-argocd.md:100]:
```suggestion
1. [Access the Longhorn UI](../../accessing-the-ui).
```

**Review by jillian-maroket — COMMENTED** [review]
@yangchiu Let me know if you have concerns about the changes.

**Comment by yangchiu** [comment]
Refined based on @jillian-maroket 's suggestions. Thanks!

File: content/docs/1.6.0/deploy/install/install-with-argocd.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+    argocd login --core
+    ```
+
+2. Set the current namespace to `argocd`:
+
+    ```bash
+    kubectl config set-context --current --namespace=argocd
+    ```
+
+3. Create the Longhorn Application custom resource.
```
[the remark this answers is not in the document]
Comment (jillian-maroket) [reply content/docs/1.6.0/deploy/install/install-with-argocd.md:34]: @yangchiu This suggestion was missed. Please apply the change so I can approve.

**Review by jillian-maroket — COMMENTED** [review]
@yangchiu One suggestion was missed. Please check.

File: content/docs/1.6.0/deploy/install/install-with-argocd.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (yangchiu) [reply content/docs/1.6.0/deploy/install/install-with-argocd.md]: Updated. Thank you!

**Review by jillian-maroket — APPROVED** [review]

**Review by innobead — APPROVED** [review]

**Pull request merged**